src/elm/Transfer.elm

```diff
@@ -45,15 +46,30 @@ type alias CommunityArgs =
 
 
 type alias Transfer =
-    { to : Eos.Name
-    , from : Eos.Name
+    { id : Int
+    , toId : Eos.Name
+    , fromId : Eos.Name
+    , to :
+        { avatar : Avatar
+        , userName : Maybe String
+        }
+    , from :
+        { avatar : Avatar
+        , userName : Maybe String
+        }
     , value : Float
     , memo : Maybe String
     , symbol : Symbol
     , blockTime : DateTime
     }
 
 
+type alias TransferAvatar =
+    { avatar : Avatar
+    , userName : Maybe String
+    }
```

Comment:
Change this to:

```elm
type alias TransferUser =
    { avatar : Avatar
    , userName : Maybe String
    , account : Eos.Name
    }
```

This will allow us to have a property in this map with all the user information we need in all the project!